web/pwa/src/assets/doc/card-browser.md

```diff
@@ -1 +1,21 @@
-The card browser allows you to manage the content of your virtual storage cards.
+# Virtual Card File Explorer
+
+**The card file explorer allows you to manage the content of your virtual storage cards.**
+
+## Information about Palm OS and file locations
+Palm OS was extended in version 4.0 with support for expansion cards. when a card is inserted, a new 'category' will appear on the Palm OS launcher (home screen) with the name of the card.
+
+Palm OS looks for files in very specific locations:
+* /Palm/Launcher -- any PRC or PDB files that are placed in this location will appear in the Palm OS launcher when the card category is selected.
+* /Palm/Programs -- most programs that enable users to store data on an expansion card will create their own folder in this location.
+* /DCIM/ -- this is the location where digital camera images are typically saved to, and as a result, most image browser programs will look in this location for photos and media.
+
+## Working with Files
```

Comment:
I think we should also add a short note on navigating with the dropdown in the menu bar. Maybe we should also document viewing and changing the file attributes. Cloudpilot purposefully ignores the readonly flag and happily deletes files that have it set, I think we should document that, too.